app/src/main/java/com/ikarabulut/Game.java

```diff
@@ -1,5 +1,7 @@
 package com.ikarabulut;
 
+import java.util.concurrent.TimeUnit;
```

Comment:
No, it was imported from when I tried it before `sleep` but I might use it now so we can use more concrete units of time measurement instead of milliseconds for `sleep`

Reply:
Oh wait this was from  `Game.java` file. No this isn't being used. I thought this was  `GameDisplay.java`